content/en/docs/concepts/architecture/nodes.md

```diff
@@ -564,7 +564,7 @@ their respective shutdown periods.
 
 {{< note >}}
 The ability to take Pod priority into account during graceful node shutdown was introduced
-as an Alpha feature in Kubernetes v1.23. In Kubernetes {{< skew currentVersion >}}
+as an Alpha feature in Kubernetes v1.23. In Kubernetes {{< skew currentPatchVersion >}}
```

Comment:
The original is fine; this statement applies to all patch versions in a minor release.